src/cell.hpp

```diff
@@ -99,6 +99,11 @@ class cell {
          }
      }
 
+    /// Return the kind of cell, used for grouping into cell_groups
+    cell_kind const get_cell_kind() const  {
```

Comment:
Should we make an abstract base class called `cell`, and make our current `cell` implementation derive from this, and have a more descriptive name like `mc_cell`, for "multicompartment cell"?